CocoaHTTPServer/CocoaHTTPServer.xcodeproj/project.pbxproj

```diff
@@ -485,7 +485,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				MACOSX_DEPLOYMENT_TARGET = 10.15;
+				MACOSX_DEPLOYMENT_TARGET = 10.13;
```

Comment:
oops, I thought I had changed this.  Must have lost the change at some point.

Reply:
oh, I see.  I had it in the framework itself set at 10.9 to silence the deprecations